start-splice-cluster

```diff
@@ -146,6 +146,8 @@ fi
 echo "Running Splice $PROFILE master and ${MEMBERS} regionservers with CHAOS = ${CHAOS} in:"
 echo "   ${RUN_DIR}"
 
+export MAVEN_OPTS="$MAVEN_OPTS -Xmx5000m"
```

Comment:
Why is that necessary?

Reply:
When compiling branch 3.0 maven ran out of memory, I just made the same change across the board.